src/pages/ProductDetail/ProductDetail.js

```diff
@@ -1,7 +1,108 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import Recommend from './component/Recommend';
+import './ProductDetail.scss';
 
 const ProductDetail = () => {
-  return <div>ProductDetail</div>;
+  const [recommendList, setRecommendList] = useState([]);
+  const [productList, setProductList] = useState({});
+  const [count, setCount] = useState(1);
+  const [modal, setModal] = useState(false);
```

Comment:
```suggestion
  const [isModalOpen, setIsModalOpen] = useState(false);
```
state의 초기값이 boolean 값이기 때문에 state의 이름을 의문형으로 지어주는 것이
해당 데이터의 값이 무엇인지 유추가 가능할 것 같네요!